bitnami/mongodb/requirements.lock

```diff
@@ -1,7 +1,6 @@
 dependencies:
```

Comment:
Sorry, this file is automatically updated by our CI/CD, please don't update it

Reply:
ok I will change it back